### ISSUE TITLE
PNotify.defaultStack not work

### ISSUE THREAD

**Issue by mahdisky** [sciactive/pnotify#327]
how to set default PNotify.defaultStack?
i did this but didnt get any result

**Comment by hperrin** [comment]
What version are you using?

**Comment by mahdisky** [comment]
v4

**Comment by hperrin** [comment]
Can you show me your code? I'm not sure what you're trying to do.

**Comment by mahdisky** [comment]
` PNotify.defaultStack = {
            "dir1": "down",
            "dir2": "left",
            "firstpos1": 25,
            "firstpos2": 25,
            "spacing1": 36,
            "spacing2": 36,
            "push": "top",
            // "context": document.body
        };`

**Comment by hperrin** [comment]
As long as it's not on one line like that (so the comment doesn't comment out the closing }, that should work. Are you running it after loading PNotify?

**Comment by mahdisky** [comment]
yes,can you send sample of this ?that's the position in top left and a little bit down

**Comment by hperrin** [comment]
Try it like this:

```js
document.addEventListener('DOMContentLoaded', function() {
  PNotify.defaultStack = { "dir1": "down", "dir2": "left", "firstpos1": 25, "firstpos2": 25, "spacing1": 36, "spacing2": 36, "push": "top", "context": document.body };
});
```
